fpga/src/main/scala/arty/IOBinders.scala

```diff
@@ -3,8 +3,7 @@ package chipyard.fpga.arty
 import chisel3._
 import chisel3.experimental.{IO}
 
-import freechips.rocketchip.util._
-import freechips.rocketchip.devices.debug._
```

Comment:
I did it partially manually. The metals plugin (for scala) in vscode indicates if an import is missing. If needed ones are removed it will scream. Removed the unused imports while converting the imports to be explicit.